packages/dina-ui/components/collection/material-sample/useMaterialSample.tsx

```diff
@@ -635,7 +627,8 @@ export function useMaterialSampleSave({
     preProcessSample
   }: PrepareSampleSaveOperationParams): Promise<SaveArgs<MaterialSample>> {
     const materialSampleInput = await prepareSampleInput(submittedValues);
-
+    const preparedBy = materialSampleInput.preparedBy || [];
+    delete materialSampleInput.preparedBy;
     const msPreprocessed =
```

Comment:
@brandonandre This 2 line of code will persist ms with preparedBy.  If I don't have line 630 and 631, it will throw exception.  That's why I copy materialSampleInput.preparedBy to a variable preparedBy.